src/components/DpCheckboxGroup/DpCheckboxGroup.vue

```diff
@@ -15,6 +15,7 @@
         text: option.label
       }"
       :name="option.name || ''"
+      :data-cy="dataCy !== '' ? dataCy + option.id : false"
```

Comment:
I think like so, If You don't define dataCy, You will get something like "undefined" + option.id. 
Isn't it enough to write it like so?
 ```suggestion
      :data-cy="dataCy + option.id"
```

Reply:
Instead of passing the colon from the parent component, you could add it here like 

```suggestion
      :data-cy="dataCy !== '' ? `${dataCy}:${option.id}` : false"
```